src/app/maps/izmir-water/aboutus/aboutus.component.less

```diff
@@ -80,7 +80,7 @@
     display: flex;
     position: relative;
     flex-direction: row;
-    justify-content: flex-start;
+    justify-content: start;
```

Comment:
This line causes a build warning: "start value has mixed support, consider using flex-start instead"